datafusion/physical-expr/src/functions.rs

```diff
@@ -752,70 +725,6 @@ mod tests {
             Int32Array
         );
         test_function!(BitLength, &[lit("")], Ok(Some(0)), i32, Int32, Int32Array);
-        test_function!(
```

Comment:
I double checked and  the functions are already tested in expr.slt (e.g. https://github.com/apache/arrow-datafusion/blob/47f4b5a67ac3b327764cbd4c0f42da7ac44854e5/datafusion/sqllogictest/test_files/expr.slt#L558-L574)

Thus removing these tests looks good to me 👍

Reply:
ok. I actually think the sql coverage is the most important (as it ensures everything works end to end). That being said having additional unit test coverage is not a bad thing either